<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategoryForm</class>
 <widget class="QMainWindow" name="CategoryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>161</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Категории расходов</string>
    </property>
    <widget class="QPushButton" name="addExpenseCategoryButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteExpenseCategoryButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QListView" name="expensesListView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="expenseCategoryLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Категории доходов</string>
    </property>
    <widget class="QListView" name="incomesListView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addIncomeCategoryButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>240</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteIncomeCategoryButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="incomeCategoryLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>CategoryForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
